### PR TITLE
Added why grommet explanation

### PR DESCRIPTION
**Pull request by alansouzati**
screen shot 2016-10-17 at 5 56 05 pm

screen shot 2016-10-17 at 5 56 11 pm

### CODE REVIEW

File: src/Home.js [hunk 1]
```diff
@@ -87,119 +101,166 @@ export default class Home extends Component {
...
+                It also offers templates and sample apps for higher order
+                component interaction.
+              </Paragraph>
+            </WhyGrommetItem>
+            <WhyGrommetItem icon={<ConfigureIcon size='xlarge' />}
+              heading='Flexible'>
+              <Paragraph align='center' margin='none'>
+                Responsive design without the complexity of managing
+                breakpoints and grids. Components are loaded individually
+                so that you only pay for what you use.
```
Comment (maysst) [inline src/Home.js]: Hi @alansouzati - you mention the word "pay" in this paragraph.  Is the intention of Grommet to be a "pay as you go" UI - i.e. **not** Open Source?  If that's **not** the case, then perhaps this is the wrong terminology?

Comment (maysst) [inline src/Home.js]: Hi @alansouzati - when you mention the word "pay" in this sentence - is the intention of Grommet to be a "pay as you go" UI - i.e. *not* Open Source?  If not, then perhaps this is the wrong terminology?

Comment (bryanjacquot) [reply src/Home.js]: Good catch @maysst, Grommet is open source, so we should be careful with this language to not give the impression there's a commercial flavor of this.  Maybe something like this:

> Components are loaded individually based on what need, making your application lightweight and efficient."

Comment (bryanjacquot) [reply src/Home.js]: Suggestion:

> Curated design and development platform with attention to detail in typography, color, space, and interaction.  More than a UI widget library for React, Grommet also offers add-ons for high order interactions.

File: src/Home.js [hunk 2]
```diff
@@ -87,119 +101,166 @@ export default class Home extends Component {
...
+                Themes allow you to stay true to your design esthetic and brand.
+              </Paragraph>
+            </WhyGrommetItem>
+            <WhyGrommetItem icon={<AccessAccessibilityIcon size='xlarge' />}
+              heading='Accessible'>
+              <Paragraph align='center' margin='none'>
+                Accessibility is a core functionality and if you use Grommet
```
Comment (bryanjacquot) [inline src/Home.js]: Suggestion:

> Accessibility is core to Grommet giving you a robust foundation for meeting accessibility standards.

File: src/Home.js [hunk 3]
```diff
@@ -87,119 +101,166 @@ export default class Home extends Component {
...
+                you are in a good path to meet the WCAG standards.
+              </Paragraph>
+              <Anchor primary={true} target='_blank'
+                href='https://vimeo.com/187068246'>Watch this</Anchor>
+            </WhyGrommetItem>
+            <WhyGrommetItem icon={<SupportIcon size='xlarge' />}
+              heading='Active'>
+              <Paragraph align='center' margin='none'>
+                Engaged community ready to assist designers and developers
```
Comment (bryanjacquot) [inline src/Home.js]: Suggestion:

> Engaged community is ready to lend a hand to new designers and developers.  The suite of designer assets, developer tools, and online training provide a seamless on-boarding experience.

**Review by bryanjacquot — COMMENTED** [review]
Looks good.  Do these suggestions help make it acceptable on the home page?

**Comment by L0ZZI** [comment]
Cleanup. I have a simplified version coming that still has the same beats.

screen shot 2016-10-18 at 8 51 01 am

**Comment by L0ZZI** [comment]
> add-ons 

...is vague inn the Comprehensive section.
What are we trying to say? extensions, tools, or extensibility?

**Comment by L0ZZI** [comment]
Cut the copy down a couple lines. And simplified some of the messaging.

screen shot 2016-10-18 at 10 11 27 am

**Comment by alansouzati** [comment]
Thanks @L0ZZI I'm going to apply these changes and update the PR. hopefully travis will not deploy it this time 😓

**Comment by alansouzati** [comment]
I've applied the changes.

File: src/Home.js [hunk 4]
```diff
@@ -87,119 +99,165 @@ export default class Home extends Component {
...
+          <Heading tag='h2'>Why Grommet?</Heading>
+          <Box direction='row' wrap={true}
+            pad={{
+              between: 'large', vertical: 'medium', horizontal: 'medium'
+            }}>
+            <WhyGrommetItem icon={<ResourcesIcon size='xlarge' />}
+              heading='Comprehensive'>
+              <Paragraph align='center'>
+                A curated design and development platform that is more than
+                just a Facebook React widget library. Grommet offers the tools
```
Comment (bryanjacquot) [inline src/Home.js]: I'd change "Facebook React" to "ReactJS"

File: src/Home.js [hunk 5]
```diff
@@ -87,119 +99,165 @@ export default class Home extends Component {
...
+                to create amazing experiences.
+              </Paragraph>
+            </WhyGrommetItem>
+            <WhyGrommetItem icon={<ConfigureIcon size='xlarge' />}
+              heading='Flexible'>
+              <Paragraph align='center'>
+                Responsive web development without the grid management,
+                theming that allows your brand to shine, and design
+                componentry that is loaded on-demand.
```
Comment (bryanjacquot) [inline src/Home.js]: "Responsive web without the hassle of grid management and modular components loaded on-demand.  Themes allow your brand to shine."

**Pull request merged**